airflow/providers/openlineage/provider.yaml

```diff
@@ -135,3 +135,11 @@ config:
         example: ~
         type: boolean
         version_added: ~
+      dag_state_change_process_pool_size:
+        description: |
+          Number of processes to utilize for processing DAG state changes
+          in an asynchronous manner within the scheduler process.
+        default: "1"
+        example: ~
+        type: integer
+        version_added: ~
```

Comment:
We should specify what version of the provider it was added

Reply:
I couldn't find it in docs but should I guess which version (probably patch) would this be released in?

Reply:
I think in #39609 we assumed next release as minor 1.8.0